search/src/Components/upload/react-images-upload/index.js

```diff
@@ -63,6 +64,8 @@ class ReactImageUploadComponent extends React.Component {
   onDropFile(e) {
     const files = e.target.files;
     const fileErrors = [];
+    const uploadedFiles = this.state.files.slice() ? this.state.files.slice() : [];
+    const filesName = uploadedFiles.map(file => file.name);
```

Comment:
This should be sufficient:

```suggestion
    const filesName = (this.state.files || []).map(file => file.name);
```
You don't need to make a copy if you're mapping it and not mutating it..